### PR TITLE
Adding three years of news

### PR DESCRIPTION
**Pull request by jhdalek55**
I've updated the "news" pages (this is the events materials and not the press page) as the last entry was from 2017

### CODE REVIEW

File: content/news.md [hunk 1]
```diff
@@ -4,6 +4,48 @@ title: News
 
 > The [press](/press) page contains a full listing of news coverage.
 
+**October 30, 2020**
+
+The Python Software Foundation live-streams a [key generation and signing ceremony](https://www.youtube.com/watch?v=jjAq7S49eow&t=3078s) that marks the first practical steps in deploying The Update Framework (TUF) to the [Python Package Index](pypi.org).  
+
+**February 15, 2020**
+
+PEP 458, Secure PyPI Downloads with Package Signing, is accepted and merged into the Python master.
```
Comment (mnm678) [inline content/news.md:13]:
```suggestion
[PEP 458](https://www.python.org/dev/peps/pep-0458/), Secure PyPI Downloads with Package Signing, is accepted and merged into the Python master.
```

**Review by mnm678 — APPROVED** [review]
I added a link for PEP 458, but otherwise, this looks great!

File: content/news.md (hunk 1 above)
Comment (jhdalek55) [reply content/news.md:13]: Thanks, Marina. I wasn't sure what was considered the official "document."

**Comment by jhdalek55** [comment]
Can someone with write access please review and merge this? Its addressing a section on the website that is really out of date.

**Review by JustinCappos — APPROVED** [review]

**Pull request merged**